tests/index.test.js

```diff
@@ -28,6 +28,7 @@ const GROUPS_HEADER_KEY = 'groups-header-key'
 const CLIENTTYPE_HEADER_KEY = 'clienttype-header-key'
 const BACKOFFICE_HEADER_KEY = 'backoffice-header-key'
 const MICROSERVICE_GATEWAY_SERVICE_NAME = 'microservice-gateway'
+const MY_REQUIRED_ENV = 'value'
```

Comment:
I would call it `MY_REQUIRED_ENV_VAR` so it's clear throughout the code it's not the whole env object

Reply:
Edited